v2/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/solarwinds/snap-plugin-lib/v2
+
+go 1.13
```

Comment:
1.15 is the current version

Reply:
but shouldn't we leave a minimal, compatible version here?

Reply:
nope. I see no reason for using such an old Go version

Reply:
Go has a 1.x compatibility promise

Reply:
discussed offline, agreed to add go versions to builder, resolving